CHANGES.txt

```diff
@@ -4,23 +4,32 @@ CHANGES
 0.19 (unreleased)
 =================
 
-- Nothing changed yet.
+* **Removed**: Removed support for Python 2.
+  
+  You have to upgrade to Python 3 if you want to use this version.
+
+* Added support for Python 3.8 and Pypy 3.6.
```

Comment:
PyPy, case spelling.

Reply:
Fixed here and also in Reg, Dectate and Importscan.